### PR TITLE
Feature/dropdown

### PR DESCRIPTION
**Pull request by pkhlopiachyi**

### CODE REVIEW

**Comment by pkucherenk0** [comment]
Approved.
Will keep in mind improvement with search.

File: src/components/Dropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+interface DropdownComponentState {
+    selected: string;
+    selectedIndex: string;
+}
+
+/**
+ *  Cryptobase Dropdown that overrides default dropdown with list of options.
+ */
+class DropdownComponent extends React.Component<DropdownComponentProps & {}, DropdownComponentState> {
+
```
Comment (josadcha) [inline src/components/Dropdown/index.tsx]: remove empty line

File: src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                </Dropdown.Item>);
+    }
+
+    private handleSelect = (elem: DropdownElem, index: number) => {
+        this.props.onSelect && this.props.onSelect(index);
+        this.setState({
+            selected: this.convertToString(elem),
+            selectedIndex: index.toString(),
+        });
+        window.console.log(this.state.selectedIndex);
```
Comment (josadcha) [inline src/components/Dropdown/index.tsx]: remove console.log

File: src/containers/Confirm/Identity/Identity.pcss [hunk 3]
```diff
@@ -37,7 +37,9 @@
         }
```
Comment (josadcha) [inline src/containers/Confirm/Identity/Identity.pcss]: convert indentation to 2 spaces

File: src/containers/Confirm/Identity/index.tsx [hunk 4]
```diff
@@ -249,8 +249,8 @@ class IdentityComponent extends React.Component<Props, IdentityState> {
                             <div className="pg-confirm__content-identity-col-row-content-label">
                                 {countryOfBirth && this.translate('page.body.kyc.identity.CoR')}
                             </div>
-                            <Dropdown
-                                className="pg-confirm__content-documents-col-row-content-number"
+                            <DropdownComponent
+                                    className="pg-confirm__content-identity-col-row-content-number-dropdown"
```
Comment (josadcha) [inline src/containers/Confirm/Identity/index.tsx]: fix indentation

**Pull request merged**